modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view_descriptive.jsp

```diff
@@ -147,7 +147,12 @@ KBArticleViewDisplayContext kbArticleViewDisplayContext = new KBArticleViewDispl
 								<span class="label-item label-item-expand"><liferay-ui:message key="expiring-soon" /></span>
 							</span>
 
-							<liferay-ui:icon-help message='<%= kbArticle.getExpirationDate()!= null ? dateFormatDateTime.format(kbArticle.getExpirationDate()) : "" %>' />
+							<clay:icon
```

Comment:
Can you extract the value of `aria-label` and `title` to a variable and reuse it? Same thing in `view_table.jsp`.

Reply:
Inlining expressions is fine as long they're "free". For example, simple getters are inlined by the JIT and transformed into direct field references; in this case, even though it is not superexpensive to inline it, it is a bit wasteful.

Reply:
That was my first approach, but later I remembered some times that after that an additional commit was added by brian inlining it.
But I will extract a variable, If you and I feel that is the correct way, we should do it.

Thanks!